scripts/release-verifier-install-gate.sh

```
@@ -0,0 +1,456 @@
+mkdir -p "$scratch_dir/python-dist"
+PIP_DISABLE_PIP_VERSION_CHECK=1 "$scratch_dir/python/bin/python" -m pip download --isolated --no-deps \
+	--only-binary=:all: --dest "$scratch_dir/python-dist" "pipelock-verify==$python_version"
+mapfile -t python_wheels < <(find "$scratch_dir/python-dist" -maxdepth 1 -type f -name '*.whl')
+if ((${#python_wheels[@]} != 1)); then
+	printf 'release verifier install gate: expected one Python wheel, found %d\n' "${#python_wheels[@]}" >&2
+	exit 2
+fi
+printf '%s  %s\n' "${python_artifact_digest#sha256:}" "${python_wheels[0]}" | sha256sum --check --status
+PIP_DISABLE_PIP_VERSION_CHECK=1 "$scratch_dir/python/bin/python" -m pip install --isolated "${python_wheels[0]}"
+
+go_reported_version="$($go_bin_dir/pipelock-verifier --version)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)